src/pcap.rs

```diff
@@ -164,6 +187,25 @@ pub fn parse_pcap_header(i: &[u8]) -> IResult<&[u8], PcapHeader, PcapError> {
                     network: Linktype(network)
                 }
             )
+        ) |
+        0x4d3c_b2a1 => do_parse!(
+            major:   be_u16 >>
+            minor:   be_u16 >>
+            zone:    be_i32 >>
+            sigfigs: be_u32 >>
+            snaplen: be_u32 >>
+            network: be_i32 >>
+            (
+                PcapHeader {
+                    magic_number: 0xd43c_b2a1,
```

Comment:
Shouldn't that be `0x4d3c_b2a1` ?
I'm asking because I'd like to factorize these `do_parse` blocks

Reply:
Yes, this looks like it was a typo on my part. Good catch :) I like the improvements you made